src/dotnet-sql-cache/project.json

```diff
@@ -1,13 +1,11 @@
 {
-  "commands": {
-    "sqlservercache": "Microsoft.Extensions.Caching.SqlConfig"
-  },
   "dependencies": {
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
     "Microsoft.Extensions.Logging": "1.0.0-*",
-    "Microsoft.Extensions.Logging.Console": "1.0.0-*"
+    "Microsoft.Extensions.Logging.Console": "1.0.0-*",
+    "Microsoft.NETCore.Platforms": "1.0.1-*"
   },
-  "description": "Contains commands for creating table and indexes in Microsoft SQL Server database to be used for ASP.NET 5 distributed caching.",
+  "description": ".NET Core tool for creating table and indexes in Microsoft SQL Server database to be used for ASP.NET 5 distributed caching.",
```

Comment:
I'm not sure if ".NET Core tool" is the right phrasing. Maybe "Tool" only or "Project tool" or something else...?